### DISCUSSION TITLE
GNPy simulation error

### DISCUSSION THREAD

**Discussion by SaiBhyri** [Telecominfraproject/oopt-gnpy#470] [Q&A] [answered]
Hi,

Getting error "Node X, needs an equalization target" with proprietary eqpt json. Runs fine with GNPy open_roadm eqpt json. Not able to figure out the cause. Any leads please?

Thanks,
Sai

**Comment by jktjkt** [comment, marked as answer]
Hi Sai, this means that GNPy wants to propagate a signal through a ROADM, but the input network description does not say how that ROADM should behave. How should the ROADM [attenuate/equalize the channels, and to which target power level](https://gnpy.readthedocs.io/en/master/json.html#equalization)? This can be set in the input JSON files, either on a [per-ROADM-model in the equipment library](https://gnpy.readthedocs.io/en/master/json.html#roadm), or in a similar way when you "instantiate" these ROADMs in the topology.
